Context/DeserializationContext.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Context;
+
+/**
+ * {@inheritdoc}
+ *
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+class DeserializationContext extends AbstractContext implements DeserializationContextInterface
+{
+    /**
+     * Constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
```

Comment:
I changed that in https://github.com/Ener-Getick/FOSRestBundle/commit/6fa9d371452e4b61d9f450a4bab3313bc6a42c27. I use constants instead.